cmd/osbuild-worker/main.go

```diff
@@ -275,14 +276,14 @@ func main() {
 			// flag to indicate all error kinds.
 			result.Success = false
 		} else {
-			log.Printf("  🎉 Job completed successfully: %s", job.Id)
+			log.Printf("  🎉 Job completed successfully: %s", token)
```

Comment:
We probably don't want to print this in the logs, do we? These tokens are not meaningful to anyone else. This can be updated in a follow-up.

Reply:
The token is not sent anymore, but maybe I should print the `location` to make it visible?

Or even send the job id?

Reply:
I've added a commit that sends the job id again, for informational purposes only. Please re-review.